test/FastaToAssembly_unit_test.py

```diff
@@ -625,6 +639,15 @@ def test_import_fasta_mass_fail_min_contig_length():
     _run_test_spec_fail(test_spec, mass=True)
 
 
+def test_import_fasta_mass_fail_invalid_obj_meta():
+    err1 = 'object_metadata must be a mapping if provided'
+    test_spec = [
+        (err1,
+         {'workspace_id': 3, 'inputs': [{'node': 'a', 'assembly_name': 'x', 'object_metadata': 'foo'}]}),
```

Comment:
Can you make a test with an empty list as well

Reply:
👍

Reply:
:+1: